listenbrainz/db/testing.py

```diff
@@ -32,16 +32,10 @@ def init_db(self):
         db.run_sql_script(os.path.join(ADMIN_SQL_DIR, 'create_indexes.sql'))
 
     def drop_tables(self):
-        with db.engine.connect() as connection:
-            connection.execute('DROP TABLE IF EXISTS "user"              CASCADE')
-            connection.execute('DROP TABLE IF EXISTS listen              CASCADE')
-            connection.execute('DROP TABLE IF EXISTS listen_json         CASCADE')
-            connection.execute('DROP TABLE IF EXISTS api_compat.token    CASCADE')
-            connection.execute('DROP TABLE IF EXISTS api_compat.session  CASCADE')
```

Comment:
I learned recently that you can use `DROP SCHEMA CASCADE` to drop all tables in a schema. As we have mostly schemas here, perhaps we could just drop the `"user"` table and `statistics` and `api_compat` schemas.
Are we still using the `listen` and `listen_json` tables??